src/Radicle/Internal/Core.hs

```diff
@@ -544,8 +555,13 @@ class ToRad t a where
 
 instance CPA t => ToRad t () where
     toRad _ = Vec Empty
+instance CPA t => ToRad t Bool where
+    toRad = Boolean
 instance (CPA t, ToRad t a, ToRad t b) => ToRad t (a,b) where
     toRad (x,y) = Vec $ toRad x :<| toRad y :<| Empty
+instance (CPA t, ToRad t a) => ToRad t (Maybe a) where
+    toRad Nothing  = Keyword (Ident "Nothing")
+    toRad (Just x) = Vec $ Keyword (Ident "Just") :<| toRad x :<| Empty
```

Comment:
Can you add `testForType` for a `Maybe` type [here](https://github.com/oscoin/radicle/blob/master/test/spec/Radicle/Tests.hs#L663)?